### DISCUSSION TITLE
Adjustable Text Size for Live Timing Window Columns

### DISCUSSION THREAD

**Discussion by justwillgreene** [f1multiviewer/issue-tracker#384] [Feature requests]
### Description:
Currently, the text size in the Live Timing window is too small for my screen, making it challenging to read the information comfortably. Therefore, my feature request is to introduce a text size adjustment option within the program's settings specifically for the columns of the Live Timing window

### Use Cases:
By allowing users to adjust the text size, the Live Timing window can accommodate individual preferences and improve readability. This would benefit users who have visual impairments or those who simply prefer larger text for better clarity. With an adjustable text size, users can effortlessly view and comprehend the Live Timing information without straining their eyes.

### Implementation Ideas:
To implement this feature, I suggest adding a text size option within the program's settings menu. Users can access this option and adjust the text size specifically for the Live Timing window. It would be ideal to follow the same size option formats already in the settings, allowing users to select the size that best suits their needs.

### Relevance to the Project:
Introducing an adjustable text size for the Live Timing window aligns with the goal of improving the user experience within the "MultiViewer for F1" project. By accommodating various text size preferences, the project can cater to a broader user base, including those with visual impairments or specific readability needs. This would also cater to a broader display base. This enhancement would contribute to the project's usability and accessibility, enhancing its value for all users.

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #194
